doc/specs/#1571 - New Tab Menu Customization/#1571 - New Tab Menu Customization.md

```diff
@@ -76,6 +76,11 @@ There are five `type`s of objects in this menu:
   - The `"entries"` property specifies a list of menu entries that will appear
     nested under this entry. This can contain other `"type":"folder"` groups as
     well!
+  - The `allowEmpty` property will force this entry to show up in the menu, even
+    if it doesn't have any profiles in it. This defaults to `false`, meaning
+    that folders without any entries in them will just be ignored when
+    generating the menu. This will be more useful with the `matchProfile` entry,
+    below.

```

Comment:
I don't quite get why this is useful in combination with `matchProfile`. `allowEmpty` feels rather niche to me.

Reply:
I imagined this as
* being useful for dynamic profiles, and shared settings files across machines. That would make the WSL entry still show up (with an <empty> entry) even when you're on a machine with no WSLs installed yet
* being a _totally optional follow-up task. P3, unimportant._

Reply:
I believe the `allowEmpty` does indeed have some merit, not only for WSL but also for other generated profiles.